CircleCI runs Windows tests under unsupported Node.js 14 - renovate fails

## Issue

Windows tests in CircleCI specified through [.circleci/config.yml](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/.circleci/config.yml) use the image `windows-server-2019-vs2019:stable` with the now unsupported Node.js `14.17.5` version.

1. renovate's PR https://github.com/cypress-io/cypress-example-kitchensink/pull/720 fails in [CircleCI Windows](https://app.circleci.com/pipelines/github/cypress-io/cypress-example-kitchensink/1586/workflows/6b791d8c-1ae1-4af2-a508-cf2e5ec4f5d5) with `npm ERR! Cannot read property 'npm-run-all' of undefined`. This PR is attempting to bump the `lockfileVersion` in [package-lock.json](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/package-lock.json) from `2` to `3`. npm [`lockfileVersion`](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json#lockfileversion) `3` is used by npm v9, however it is only backwards compatible to npm v7. Node.js `14.17.5` is configured with the earlier version npm `6.14.14` thus causing a compatibility issue.
2. Node.js `14` entered [end-of-life](https://github.com/nodejs/release#release-schedule) on April 30, 2023.
3. [.node-version](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/.node-version) specifies Node.js `18` as the required version for this repo.

## Suggestion

Update [.circleci/config.yml](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/.circleci/config.yml) to run under Node.js `18` (currently the LTS version) for the Windows tests.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -9,19 +9,13 @@ version: 2.1
 orbs:
   # use Cypress orb from CircleCI registry
   cypress: cypress-io/cypress@3.1.4
+  win: circleci/windows@5.0.0
 
 executors:
   mac:
     macos:
       # Executor should have Node >= required version
       xcode: "14.0.0"
-  win:
-    # copied the parameters from
-    # https://circleci.com/developer/orbs/orb/circleci/windows
-    machine:
-      image: "windows-server-2019-vs2019:stable"
-      resource_class: "windows.medium"
-      shell: "bash.exe"
   browsers:
     docker:
       - image: 'cypress/browsers:node-18.15.0-chrome-111.0.5563.146-1-ff-111.0.1-edge-111.0.1661.62-1'
@@ -29,7 +23,10 @@ executors:
 jobs:
   win-test:
     working_directory: ~/app
-    executor: win
+    executor:
+      name: win/default
+      size: medium
+      shell: bash.exe
     steps:
       - checkout
 
@@ -66,7 +63,10 @@ jobs:
 
   win-test-chrome:
     working_directory: ~/app
-    executor: win
+    executor:
+      name: win/default
+      size: medium
+      shell: bash.exe
     steps:
       - checkout
 
@@ -75,7 +75,7 @@ jobs:
 
       # install Chrome browser on Windows machine using Chocolatey
       # https://chocolatey.org/packages/GoogleChrome
-      - run: choco install googlechrome --ignore-checksums
+      - run: choco install googlechrome --ignore-checksums -y
       - run: npm ci
       - run: npm run cy:verify
       - run: npm run cy:info
@@ -106,7 +106,10 @@ jobs:
 
   win-test-firefox:
     working_directory: ~/app
-    executor: win
+    executor:
+      name: win/default
+      size: medium
+      shell: bash.exe
     steps:
       - checkout
 
@@ -115,7 +118,7 @@ jobs:
 
       # install Firefox browser on Windows machine using Chocolatey
       # https://chocolatey.org/packages/Firefox
-      - run: choco install firefox
+      - run: choco install firefox -y
       - run: npm ci
       - run: npm run cy:verify
       - run: npm run cy:info

